reco_utils/nni/nni_utils.py

```diff
@@ -0,0 +1,58 @@
+import json
+import numpy as np
+import os
+import requests
+import time
+
+NNI_REST_ENDPOINT = 'http://localhost:8080/api/v1/nni'
```

Comment:
@anargyri, one question, all these utils are for supporting the code or for the tests?

Reply:
They enable running the code automatically from the notebook.

Reply:
ok cool